### PR TITLE
Confirmation and better copy for repair default

### PR DESCRIPTION
**Pull request by camillebaldock**
Following these changes: https://github.com/heroku/shogun/pull/7535

### CODE REVIEW

File: commands/credentials/repair_default.js [hunk 1]
```diff
@@ -8,12 +8,17 @@ function * run (context, heroku) {
...
 
-  const {app, args} = context
+  const {app, args, flags} = context
 
   let db = yield fetcher.addon(app, args.database)
   if (util.starterPlan(db)) throw new Error('This operation is not supported by Hobby tier databases.')
 
-  yield cli.action(`Resetting permissions for default role to factory settings`, co(function * () {
+  yield cli.confirmApp(app, flags.confirm, `WARNING: Destructive Action
+This command will give the default credential ownership of all database objects owned by additional credentials.
```
Comment (msakrejda) [inline commands/credentials/repair_default.js]: Should we be more explicit that it's removing ownership from the other credentials? I.e., something like "Ownership of all database objects owned by additional credentials will be transferred to the default role"?

File: commands/credentials/repair_default.js [hunk 2]
```diff
@@ -8,12 +8,17 @@ function * run (context, heroku) {
...
-  const {app, args} = context
+  const {app, args, flags} = context
 
   let db = yield fetcher.addon(app, args.database)
   if (util.starterPlan(db)) throw new Error('This operation is not supported by Hobby tier databases.')
 
-  yield cli.action(`Resetting permissions for default role to factory settings`, co(function * () {
+  yield cli.confirmApp(app, flags.confirm, `WARNING: Destructive Action
+This command will give the default credential ownership of all database objects owned by additional credentials. 
+It will also grant the default credential admin option all additional credentials.
```
Comment (msakrejda) [inline commands/credentials/repair_default.js]: I think you accidentally a word here.

**Comment by camillebaldock** [comment]
@uhoh-itsmaciek how's it looking now? :)

**Review by msakrejda — APPROVED** [review]
+1

**Pull request merged**